src/targets/gpu/fuse_ops.cpp

```diff
@@ -152,6 +153,12 @@ struct fusion
     }
 };
 
+std::vector<std::string> get_supported_archs()
```

Comment:
Actually instead of a std::vector, can you use a std::set<std::string> instead? And then I believe you can change the logic to use `if contains(supported_archs, device_name)` for L173